stable/neo4j/templates/core-statefulset.yaml

```diff
@@ -24,6 +24,8 @@ spec:
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
         env:
+          - name: NEO4J_ACCEPT_LICENSE_AGREEMENT
+            value: "yes"
```

Comment:
I think @moxious was looking at this and trying to think of a good way to do this...

Reply:
I'd be interested in any ideas, the choices I saw were:
1. Switch to using the community image 
2. Force users to set a variable to accept the license
3. Accept it by default. 
Since in general we want `helm install stable/neo4j` to just work, I chose to go this route, but totally understand why it may not be optimal, open to ideas/suggestions here.

Reply:
I'd use option two. Leave it empty and use the `required` function. Provide a custom values file that sets it for CI only.